### PR TITLE
Somewhat cursed lateload z-level minimap hard del fix

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Fucking guh. Fixes the funny hard del.

Please someone suggest a less rediculous way to do this.

### CODE REVIEW

**Comment by spookydonut** [comment]
a bit weird but it works

**Review by spookydonut — APPROVED** [review]

File: code/controllers/subsystem/minimaps.dm [hunk 1]
```diff
@@ -226,6 +226,7 @@ SUBSYSTEM_DEF(minimaps)
 		if(!(earlyadds["[target.z]"]))
 			earlyadds["[target.z]"] = list()
 		earlyadds["[target.z]"] += CALLBACK(src, PROC_REF(add_marker), target, hud_flags, blip)
+		RegisterSignal(target, COMSIG_QDELETING, PROC_REF(remove_earlyadd), override = TRUE) //if it was loaded before the z-level
```
Comment (TiviPlus) [inline code/controllers/subsystem/minimaps.dm:229]: This never gets unregistered if its processed normally is that intentional

Comment (TiviPlus) [reply code/controllers/subsystem/minimaps.dm:229]: Also you need to write a detailed explanation for using override = true

Comment (Lumipharon) [reply code/controllers/subsystem/minimaps.dm:229]: when the zlevel finished loading, the signal gets overridden

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**